tests/FakerDotNet.Tests/FakerContainerTests.cs

```diff
@@ -80,5 +80,11 @@ public void Zelda_returns_IZeldaFaker()
         {
             Assert.IsInstanceOf<IZeldaFaker>(_fakerContainer.Zelda);
         }
+
+        [Test]
+        public void Friends_returns_IFriendsFaker()
```

Comment:
Same as above. These tests need to be in the same position as the properties on the class (in alphabetical order).